Index: src/main/java/com/example/demo/Tools/ConnexionBDD.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.demo.Tools;\r\n\r\n\r\nimport java.sql.Connection;\r\nimport java.sql.DriverManager;\r\nimport java.sql.SQLException;\r\nimport java.util.TimeZone;\r\n\r\npublic class ConnexionBDD\r\n{\r\n    private static Connection cnx;\r\n\r\n    public ConnexionBDD() throws ClassNotFoundException, SQLException {\r\n        //String pilote = \"com.mysql.jdbc.Driver\";\r\n        String pilote = \"com.mysql.cj.jdbc.Driver\";\r\n        // chargement du pilote\r\n        Class.forName(pilote);\r\n        // L'objet connexion à la BDD avec le nom de la base, le user et le password\r\n        cnx = DriverManager.getConnection(\"jdbc:mysql://localhost/HelpOrt?useSSL=true&serverTimezone=\"\r\n                + TimeZone.getDefault().getID(), \"root\", \"\");\r\n    }\r\n    public static Connection getCnx() {\r\n        return cnx;\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/demo/Tools/ConnexionBDD.java b/src/main/java/com/example/demo/Tools/ConnexionBDD.java
--- a/src/main/java/com/example/demo/Tools/ConnexionBDD.java	(revision 418f530f6c996721f1edd0033057b18d1ab60749)
+++ b/src/main/java/com/example/demo/Tools/ConnexionBDD.java	(date 1703171146431)
@@ -16,7 +16,7 @@
         // chargement du pilote
         Class.forName(pilote);
         // L'objet connexion à la BDD avec le nom de la base, le user et le password
-        cnx = DriverManager.getConnection("jdbc:mysql://localhost/HelpOrt?useSSL=true&serverTimezone="
+        cnx = DriverManager.getConnection("jdbc:mysql://localhost/HelpOrtJ?useSSL=true&serverTimezone="
                 + TimeZone.getDefault().getID(), "root", "");
     }
     public static Connection getCnx() {
